d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -26,21 +27,27 @@ def on_market_cycle(self, market_info):
         if self.is_finished is True:
             return
 
-        market_uuid = self.get_uuid_from_area_name("Market")
+        market_uuid = self.get_uuid_from_area_name("House")
         for area_uuid, area_dict in self.latest_grid_tree_flat.items():
-            if "asset_info" not in area_dict or area_dict["asset_info"] is None:
-                continue
-
             logging.info(
-                f"current_market_fee: {self.grid_fee_calculation.calculate_grid_fee(area_uuid, market_uuid)}")
-            if key_in_dict_and_not_none_and_greater_than_zero(area_dict, "available_energy_kWh"):
-                self.add_to_batch_commands.offer_energy(area_uuid=area_uuid, price=1,
-                                                        energy=area_dict["asset_info"]["available_energy_kWh"] / 2) \
-                    .list_offers(area_uuid=area_uuid)
-            if key_in_dict_and_not_none_and_greater_than_zero(area_dict, "energy_requirement_kWh"):
-                self.add_to_batch_commands.bid_energy(area_uuid=area_uuid, price=30,
-                                                      energy=area_dict["asset_info"]["energy_requirement_kWh"] / 2) \
-                    .list_bids(area_uuid=area_uuid)
+                f"current_market_fee: "
+                f"{self.grid_fee_calculation.calculate_grid_fee(area_uuid, market_uuid)}")
+            if "asset_info" not in area_dict or area_dict["asset_info"] is None:
+                if area_uuid == market_uuid:
+                    self.add_to_batch_commands.last_market_dso_stats(area_uuid=area_uuid). \
+                        grid_fees(area_uuid=area_uuid, fee_cents_kwh=5)
+            else:
+                if key_in_dict_and_not_none_and_greater_than_zero(area_dict["asset_info"],
```

Comment:
Just wanted to mention that `0` is considered to have a `False` value. So `key_in_dict_and_not_none_and_greater_than_zero` would be the same as evaluating `.get()`:
```python
area_dict = {'asset_info': {'available_energy_kWh': 0}}
if area_dict['asset_info'].get('available_energy_kWh'):
     print('true!')
else:
    print('false!')

false!
```
and
```python
area_dict = {'asset_info': {'available_energy_kWh': 1}}
if area_dict['asset_info'].get('available_energy_kWh'):
     print('true!')
else:
    print('false!')

true!
```